docs/modules/help/pages/reporting_servers.adoc

```diff
@@ -0,0 +1,24 @@
+= Accessing the Servers Report
+
+WARNING: Depending on your access rights, your actual screen might differ from what is shown in the screenshots in this help system.
+
+[[overview]]
+== Overview
+
+This screen gives you an overview of all the servers currently managed on your behalf.
```

Comment:
This screen probably needs more help as it's not straight-forward. One has to navigate to one server under server management, select the facts tab and choose there which facts to include in this report.

Reply:
@akosma Can you improve this page a bit? Maybe @davidgubler can help you getting to know this screen better.

Reply:
Please check the current description, including more images and detailed step-by-step instructions.

Reply:
Perfect! Thanks.